### ISSUE TITLE
The navbar is too wide, and causes issues on mobile

### ISSUE THREAD

**Issue by ArendPeter** [Equal-Vote/bettervoting#704]
When I open the webpage on mobile, and I can scroll the page to the right (which should happen)

![image](https://github.com/user-attachments/assets/f038f6af-d540-441a-8e76-ea67cf7f7abb)

**Comment by ArendPeter** [comment]
This could also be the root cause of the following issue:
![image](https://github.com/user-attachments/assets/4c5a1975-3f59-4c4a-b4a0-e7632e87c455)

**Comment by fsargent** [comment]
I don't think this is an issue anymore, it works great on the mobile situation.
![CleanShot 2024-12-22 at 12 19 48@2x](https://github.com/user-attachments/assets/c6df536f-2e62-4aa4-9f52-d99198dc64a9)

![CleanShot 2024-12-22 at 12 20 33@2x](https://github.com/user-attachments/assets/f10677f3-03cf-4d11-b303-633eeb3066e4)

**Comment by ArendPeter** [comment]
We've been trying to make it responsive down to 320px. Looks like it still breaks at that size

That said 320px might be overkill, so I think we can keep this as a lower priority item

**Comment by waugh** [comment]
When I am logged in, only part of the person icon shows. I can select it in simulation, but don't know whether it would work on an actual 320px-wide device.

![Image](https://github.com/user-attachments/assets/45d2846f-5ec8-49ee-a6a4-321f0d42ac8a)

**Comment by jacksonloper** [comment]
This is still pretty suboptimal on my S25 phone.  It isn't unusable, but it is ugly.
